src/main/drivers/pwm_output.c

```diff
@@ -544,7 +544,7 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t motorCount, uint8_t command, bo
                 if (index == i || index == ALL_MOTORS) {
                     commandControl->command[i] = command;
                 } else {
-                    commandControl->command[i] = command;
+                    commandControl->command[i] = 0;
```

Comment:
`DSHOT_CMD_MOTOR_STOP` will make it clearer what is happening here.

Reply:
Well we're not really trying to send a motor stop command. Rather skipping the command for that motor. Maybe we define a `DSHOT_CMD_NONE 255` and make the logic look for this to skip the motor.  Although I'm not sure `DSHOT_CMD_MOTOR_STOP` is something we'd ever want to send as a DSHOT command.

Reply:
So we are filtering out the 0 values in the array lower down in the driver? I thought we were just sending them to the ESCs, which translates to sending 'motor stop' commands.

Reply:
Ah, actually you're right. I saw this and thought it was skipping the entire command but it's only driving whether to set the telemetry bit:
```
    if (pwmDshotCommandIsProcessing()) {
        value = pwmGetDshotCommand(index);
#ifdef USE_DSHOT_TELEMETRY
        // reset telemetry debug statistics every time telemetry is enabled
        if (value == DSHOT_CMD_SIGNAL_LINE_CONTINUOUS_ERPM_TELEMETRY) {
            dshotInvalidPacketCount = 0;
            readDoneCount = 0;
        }
#endif
        if (value) {
            motor->requestTelemetry = true;
        }
```
Then I guess setting it to `DSHOT_CMD_MOTOR_STOP` is fine. In the big picture it's kind of limiting as it restricts all DSHOT commands to only be used in the motor stopped state (or risk stopping the motors).  For our current uses I guess it doesn't matter.

Reply:
Updated to use `DSHOT_CMD_MOTOR_STOP` instead of `0`.

Reply:
Yes it is limiting, but the changes required to make it possible to completely skip individual ESCs when sending Dshot commands would be rather far reaching (and probably require precautions to avoid ESC disarms caused by timeout), so I am not sure it's a good idea to implement them (it's been looked at in the past) until we've got an actual use case for it.

Using `DSHOT_CMD_MOTOR_STOP` will at least make it obvious in this layer what is happening.